lib/cocoapods/command/init.rb

```diff
@@ -86,19 +86,11 @@ def target_module(app, tests)
 
         app_uses_swift = app.source_build_phase.files_references.any? { |fr| File.extname(fr.path) == '.swift' }
 
-        target_module << if app_uses_swift
-                           <<-RUBY
-  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+        target_module << <<-RUBY
+  # Comment the following line to use static libraries. With static libraries, the startup time is better, and with dynamic libraries, the incremental compilation time is better.
```

Comment:
I am not sure the last part is true, why is incremental compilation better? I think "# Comment the following line to use static libraries." is good enough here.

Reply:
Incremental compilation is better because, when it's linking the main executable, it won't have to link in the cocoapods, those cocoapods are already linked into their own dynamic frameworks. I am fine with keeping or removing it. I could leave out the bit about startup time as well, although in practice I see many people that have `use_frameworks` in their Podfile and don't understand that it increases startup time

Reply:
+1 on keeping it short.  

I agree with the incremental compilation point in theory. However in practice I would suspect it is negligible.  

I think the nuances are too complicated to try to capture here and providing summary info that let's people know what to search for is sufficient.

Reply:
The incremental compilation point is measurable and it can have a big impact on big projects, e.g. I've worked on a project with a 25-second link time due to how many files were being included, and which would've no doubt become much smaller using dylibs. But I do understand that most projects don't have to worry about that. I'm perfectly fine removing it.

As for removing the bit about startup time, I don't think that most iOS devs search things like "static vs. dynamic libraries", and even if they did, most of the articles don't really discuss startup time. If we really don't want to mention it, perhaps we could link to some doc on an overview of static and dynamic libraries, or comment out `use_frameworks!` (and add `use_modular_headers`) by default. Otherwise, we are essentially accepting that most iOS devs will continue to be ignorant of this.

Reply:
But this sounds like that its only affected by big projects which I would expect people will be able to perform such an analysis. I dont think we want to provide this messaging for the majority of the projects which are small and this improvement is negligible.

Reply:
Even small projects can pull in lots of code via Cocoapods. But, I'd be fine with a short comment that doesn't mention any of the tradeoffs. Ideally in that case, I'd like to comment out use_frameworks, but I could live with leaving it as it is.

Reply:
SG, can you please update the PR to reflect the shorter comment? Then we can merge!